raiden/network/proxies/token_network.py

```diff
@@ -284,13 +291,14 @@ def _new_netting_channel(
             settle_timeout=settle_timeout,
         )
         if not gas_limit:
+            newest_block = self.client.blockhash_from_blocknumber(checking_block)
             self.proxy.jsonrpc_client.check_for_insufficient_eth(
                 transaction_name="openChannel",
                 transaction_executed=False,
                 required_gas=self.gas_measurements["TokenNetwork.openChannel"],
-                block_identifier=checking_block,
+                block_identifier=newest_block,
             )
-            self._new_channel_postconditions(partner=partner, block=checking_block)
+            self._new_channel_conditions(partner=partner, block=newest_block)
```

Comment:
pre and post conditions are not the same, post conditions are to handle data races, please fix.